admin/publish-release.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+
+set -e
+
+# Publish to BinTray if the HEAD commit is tagged with a version number.
+if [ "$PULL_REQUEST" != "false" ]; then
+  echo "Shippable is building a pull request, not publishing."
+  echo "PULL_REQUEST is equal to $PULL_REQUEST"
+  exit 0
+fi
+
+if [ "$BRANCH" != "master" ]; then
+  echo "Shippable is building on branch $BRANCH, not publishing."
+  echo "BRANCH is equal to $BRANCH"
+  exit 0
+fi
+
+numParents=`git log --pretty=%P -n 1 | wc -w | xargs`
```

Comment:
A comment on this would be helpful (mostly the `xargs` hack). I think `tr -d ' '` might work, too.